crates/modelardb_server/src/main.rs

```diff
@@ -44,7 +44,10 @@ static ALLOC: snmalloc_rs::SnMalloc = snmalloc_rs::SnMalloc;
 
 /// The port of the Apache Arrow Flight Server. If the environment variable is not set, 9999 is used.
 pub static PORT: Lazy<u16> = Lazy::new(|| match env::var("MODELARDBD_PORT") {
-    Ok(port) => port.parse().unwrap(),
+    Ok(port) => port
+        .parse()
+        .map_err(|_| "MODELARDBD_PORT must be a port from 1 to 65535 if set")
```

Comment:
This error message is missing punctuation. Maybe also change it to `MODELARDBD_PORT must be between 1 and 65535.` or something similar.

Reply:
Fixed.